.github/get-modified-packages.php

```diff
@@ -12,6 +12,11 @@
 $allPackages = json_decode($_SERVER['argv'][1], true, 512, \JSON_THROW_ON_ERROR);
 $modifiedFiles = json_decode($_SERVER['argv'][2], true, 512, \JSON_THROW_ON_ERROR);
 
+// Sort to get the longest name first (match bridge not component)
+usort($allPackages, function($a, $b) {
+    return strlen($b) - strlen($a);
```

Comment:
`strlen($b) <=> strlen($a) ?: $a <=> $b`?

Reply:
Why `?: $a <=> $b`?

Reply:
to sort packages in alpha order when they have the same length

Reply:
Hehe, yeah, sure. 

But it does not matter too much.. But alphabetically is better than possibly random.